source/guides/ember-cli/index.md

```diff
@@ -0,0 +1,27 @@
+Ember CLI has become a major part of the Ember experience.
```

Comment:
I'd shrink this page down to a single section which lives on the installing page (where the node section is now) and just give a brief reasoning on why Ember uses a CLI:
> Ember comes with a toolchain called Ember CLI that provides everything you need while developing your app behind a singe command: `ember`.
> - State of the art asset management (use `ember build` to build your entire project)
> - Quick scaffolding like `ember generate route some-page`
> - ES6 modules to keep your project organized
> - A complete testing framework (`ember test`)
> - [Addons](http://emberaddons.com) to share or use libraries from other Ember developers